lib/addons/cloudwatch-adot-addon/index.ts

```diff
@@ -29,7 +29,17 @@ export interface CloudWatchAdotAddOnProps {
      * Name to deploy the ADOT Collector for CloudWatch.
      * @default 'adot-collector-cloudwatch'
      */
-     name?: string;
+    name?: string;
+    /**
+     * Metrics name selectors to write to CloudWatch.
+     * @default "['apiserver_request_.*', 'container_memory_.*', 'container_threads', 'otelcol_process_.*']"
+     */
+    metricsNameSelectors?: string[];
+    /**
+     * Pod Labels to select your application pods emitting custom metrics.
+     * @default 'frontend|downstream(.*)'
+     */
+    podLabels?: string;
```

Comment:
Can you be more specific what this means? Are these labels on pods? If so, why would we add frontend|downstream here? Choices on the demo apps should not drive the default that we apply.